models/unstructured/rag_hubspot__document.sql

```diff
@@ -29,7 +29,7 @@ final as (
             "grouped.comments_group_markdown"]) }}
             as chunk
     from deal_document
-    join grouped
+    inner join grouped
```

Comment:
It looks like in PR #24 @levonkorganyan is proposing to change this to a `left` join instead. Let's sync to make sure we're making the right update.

Reply:
Bumping this message as this is the only other remaining question in this PR before approval.

Reply:
pinged Levon in his PR